src/main/java/com/google/api/codegen/transformer/MockServiceTransformer.java

```diff
@@ -73,4 +106,244 @@
     }
     return mocks;
   }
+
+  public List<MockServiceUsageView> createMockServices(
+      SurfaceNamer namer, Model model, ApiConfig apiConfig) {
+    List<MockServiceUsageView> mockServices = new ArrayList<>();
+
+    for (Interface service : getGrpcInterfacesToMock(model, apiConfig)) {
+      MockServiceUsageView mockService =
+          MockServiceUsageView.newBuilder()
+              .className(namer.getMockServiceClassName(service))
+              .varName(namer.getMockServiceVarName(service))
+              .implName(namer.getMockGrpcServiceImplName(service))
+              .registerFunctionName(namer.getServerRegisterFunctionName(service))
+              .build();
+      mockServices.add(mockService);
+    }
+
+    return mockServices;
+  }
+
+  // TODO(shinfan): Convert to use TestMethodView.
+  public GapicSurfaceTestCaseView createTestCaseView(
```

Comment:
@pongad do you mind taking care of this now or I can fix this TODO after this PR is merged. I am good with either way.